scripts/kfctl.sh

```diff
@@ -15,9 +15,14 @@ WHAT=$2
 
 ENV_FILE="env.sh"
 
+KUBEFLOW_VERSION=${KUBEFLOW_VERSION:-"master"}
```

Comment:
Why do you need KUBEFLOW_VERSION?

Reply:
To download the appropriate repository archive.

Reply:
This needs to be set here, because createEnv is called after downloading corresponding shell scripts that are needed by kfctl.sh.